processing/src/main/java/io/druid/query/extraction/ExtractionFn.java

```diff
@@ -71,13 +74,9 @@
    */
   public String apply(Object value);
 
-  public String apply(String value);
```

Comment:
I've missed tagging `discuss` label on this. Yes, we should consider it but I've just wanted to show the intention.